README.md

```diff
@@ -33,6 +33,7 @@ apply plugin: 'com.google.cloud.tools.endpoints-framework-server'
 The plugin exposes the following server side goals
 * `endpointsClientLibs` - generate client libraries
 * `endpointsDiscoveryDocs` - generate discovery documents
+* `endpointsOpenApiDocs` - generate openapi documents
```

Comment:
Docs will not change until a release. I'll make this update when we do the release.